lib/parsers/mp3_parser.rb

```diff
@@ -315,5 +320,5 @@ def with_id3tag_local_configs
     end
   end
 
-  FormatParser.register_parser new, natures: :audio, formats: :mp3, priority: 99
+  FormatParser.register_parser new, natures: :audio, formats: :mp3, priority: 101
```

Comment:
Just for the record: which other parsers may be impacted by the change of priority here? As far as I know, the MP3 parser was at the end of the line at some point in the past, were there other parsers added after it? And if so, would the ordering mess up them?

Reply:
The order for the remaining parsers doesn't change, aside from pushing MP3 to the end.

Most parsers did not have a set priority, so they defaulted to 99 on registration – the same as MP3 which had priority 99 hardcoded. Here is what the state of priorities looked like before my change:
![image](https://github.com/WeTransfer/format_parser/assets/2901534/edf076ed-e92d-4b22-b342-a132584a8b0d)
all those marked with red are defaulting to 99...

Reply:
Thanks for the details! And when they have the same prio, I would assume there's no definite ordering, right?

I'm just thinking whether anything that was after MP3s was kinda depending on this parser and now their behaviour might change 🤔 

If it's a non-issue, feel free to ignore 😊

Reply:
It doesn't seem to be a big issue. When they have the same priority they end up in the order that they got registered. That will be the same every time unless we change something. In the future we might want to specify priorities to all of them. But for now after moving MP3 at the bottom, the tests seem to confirm that all parsers continue to function as expected.

Reply:
Awesome, thanks for confirming!